src/test/java/org/fundacionjala/pivotal/cucumber/runner/RunCucumberTest.java

```diff
@@ -1,7 +1,11 @@
 package org.fundacionjala.pivotal.cucumber.runner;
 
 import cucumber.api.CucumberOptions;
+import cucumber.api.java.After;
 import cucumber.api.testng.AbstractTestNGCucumberTests;
+import org.fundacionjala.core.ui.WebDriverManager;
+import org.openqa.selenium.WebElement;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.WebElement'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)